### PR TITLE
Feature/99 fix project create

### PR DESCRIPTION
**Pull request by Brend-Smits**
Fix issue where creating a project would always result in project being assigned to user with Id 1. 
Closes: #99

### CODE REVIEW

File: API/Controllers/ProjectController.cs [hunk 1]
```diff
@@ -153,12 +150,12 @@ public async Task<IActionResult> UpdateProject(int projectId, [FromBody] Project
...
         public async Task<IActionResult> DeleteProject(int projectId)
         {
-            if(await _projectService.FindAsync(projectId) == null)
+            if(await projectService.FindAsync(projectId) == null)
             {
                 return NotFound();
             }
-            await _projectService.RemoveAsync(projectId);
-            _projectService.Save();
+            await projectService.RemoveAsync(projectId);
```
Comment (wotwot563) [inline API/Controllers/ProjectController.cs:157]: does not validate if the user may delete

**Comment by Brend-Smits** [comment]
Blocked until we implement a way for the API to retrieve user id from access_token even if the user authenticates via Swagger or Postman

**Comment by Brend-Smits** [comment]
Blocked by: https://github.com/DigitalExcellence/dex-backend/pull/102

**Comment by Brend-Smits** [comment]
Closing in favor of https://github.com/DigitalExcellence/dex-backend/pull/112

**Pull request closed without merging**